### DISCUSSION TITLE
How do I set the font to show while not typing?

### DISCUSSION THREAD

**Discussion by SoundCrafterYT** [Earu/EasyChat#98] [General]
I can't set the font to be shown when not typing into chat.
I swear there was as setting that could preview the fonts and change outline etc.

**Comment by Earu** [comment]
Hi, there is indeed settings for that if you open the chatbox, then click on the gear `⚙️` at the rop right corner of the chat window, then go in `chat HUD` and then click `Font Editor`.

![image](https://user-images.githubusercontent.com/20880060/192140846-38b2376c-cf0b-458c-a3a7-b60f13390920.png)
